src/Umbraco.Core/Models/Editors/ContentPropertyData.cs

```diff
@@ -25,10 +25,27 @@ public ContentPropertyData(object? value, object? dataTypeConfiguration)
     /// </summary>
     public object? DataTypeConfiguration { get; }
 
+    /// <summary>
+    ///     Gets or sets the content owning the property.
+    /// </summary>
+    public IContentBase? Content { get; set; }
```

Comment:
When this is nullable it can lead to all kind of wierd things. 

Could this be non-nullable?

Reply:
I guess it can be? I made it nullable since I didn't want to impose required fields, but it's weird a CPD can exist without an "owning" Content, right? It also solves the comment above where things explode if it would be null.